tests/model/model.cpp

```diff
@@ -624,3 +624,59 @@ TEST(Model, removeComponentInsensitiveToOrder)
     EXPECT_EQ(size_t(0), modelParsed->componentCount());
     EXPECT_EQ(size_t(2), modelApi->componentCount());
 }
+
+TEST(Model, cleanModel)
+{
+    // Make a model with empty components, unrequired units, empty import sources.
+    auto e = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
```

Comment:
Replace `auto e` with `const std::string e`. Same elsewhere in this PR.

Reply:
f15e055d